cpp/cmake/thirdparty/get_rmm.cmake

```diff
@@ -32,8 +32,8 @@ function(find_and_configure_rmm VERSION)
         INSTALL_EXPORT_SET  cugraph-exports
         CPM_ARGS
             GIT_REPOSITORY  https://github.com/rapidsai/rmm.git
-            GIT_TAG         branch-${MAJOR_AND_MINOR}
-            GIT_SHALLOW     TRUE
+            GIT_TAG         23bbe745af1d988224b5498f7b8e3fe3720532d4
+            GIT_SHALLOW     FALSE
```

Comment:
Can a `FIXME` be added somewhere so we remember to set `GIT_SHALLOW` back to `TRUE` when we eventually go back to using a branch name again?  I think we'll want this to minimize the cloning overhead by not getting the entire git history when set back to `TRUE`.